code/datums/supply_packs/food.dm

```diff
@@ -84,13 +213,34 @@
 					/obj/item/reagent_container/food/snacks/grown/wheat,
 					/obj/item/reagent_container/food/snacks/grown/wheat,
 					)
-	cost = 10
+	cost = 7
```

Comment:
defines not numbers

Reply:
Define for cost got remove in a previews PR with oversight of fira thepiachu and HarryOb.
https://github.com/cmss13-devs/cmss13/pull/1568
del: remove old define RO_PRICE and directly input the cost variable number for crates.